src/systems/brush.js

```diff
@@ -207,7 +218,7 @@ AFRAME.registerSystem('brush', {
       }
     }
   },
-  addNewStroke: function (brushName, color, size) {
+  addNewStroke: function (brushName, color, size, owner='local', timestamp=Date.now()) {
```

Comment:
For consistency I would do `var owner = owner || 'local'` instead of using default parameter values